src/ambient.d.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/ban-types */
+// Squelch warnings of image imports from your assets dir
+
+type AssetData = Object[];
```

Comment:
Can you make this an [opaque newtype](https://blog.beraliv.dev/2021-05-07-opaque-type-in-typescript)?
```suggestion
declare const _opaque = unique symbol;
type Opaque<T, V> = T & { readonly [_opaque]: V }

type AssetData = Opaque<Object[], "AssetData">;
```

Reply:
Sure